docs/serving/using-auto-tls.md

```diff
@@ -19,41 +44,31 @@ provisioning:
   - [Knative Serving](../install/).
   - [Istio with SDS, version 1.1 or higher](../install/installing-istio.md#installing-istio-with-SDS-to-secure-the-ingress-gateway) or [Gloo, version 0.18.16 or higher](../install/Knative-with-Gloo.md).
```

Comment:
Is it necessary to specify versions or can it just be something more like 'latest version'?
Mentioning specific versions means this will need to be maintained / updated going forward to ensure the docs stay up to date.